source/upgrade.txt

```diff
@@ -170,10 +179,11 @@ The v4.0 driver introduces the following breaking changes to the build system:
 - Drops support for the ``MONGOCXX_OVERRIDE_DEFAULT_INSTALL_PREFIX`` configuration
   option. Default CMake behavior for ``CMAKE_INSTALL_PREFIX`` is now respected.
 
-- Sets the ``ENABLE_TESTS`` configuration option to ``OFF`` by default. To
-  re-enable building test targets, you must set ``ENABLE_TESTS`` to ``ON``.
-  To include test targets in the "all" target, you must also set the
-  ``BUILD_TESTING`` option to ``ON``.
+.. note:: 
```

Comment:
I: I think we should still leave this as a v4.0 change, since that release did set ``ENABLE_TESTS`` configuration option to ``OFF`` by default - the only time it didn't was if you auto-downloaded the C driver. and (from my understanding) that's the specific case the 4.1 release fixed

Reply:
This is correct. The bugfix applies specifically to auto-downloaded C Driver configurations.